api/bag_vrrefinements.cpp

```diff
@@ -321,7 +336,14 @@ void VRRefinements::writeProxy(
             throw DatasetNotFound{};
 
         auto pDataset = this->getDataset().lock();
+        // TODO: Confirm that this is what we want --- this resets the dimensions of the
```

Comment:
@brian-r-calder The code in this branch needs to be updated to account for there now being two dimensions in the file data space (see [here](https://github.com/OpenNavigationSurface/BAG/blob/109-LayerDescriptor-shape-parameters/api/bag_surfacecorrections.cpp#L651) for example). However, having this modify the main BAG dataset dimensions seems wrong. I'm not sure what the right answer is though.